doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -676,92 +740,108 @@ fantray0    N/A  fantray0.fan      55%       intake     Present        OK  20230
 fantray1    N/A  fantray1.fan      56%       intake     Present        OK  20230728 06:41:17
 ```
 
-#### 3.4.1 Reboot Cause
-* There are two CLIs "show reboot-cause" and "show reboot-cause history" which are applicable to both DPUs and the Switch. However, when executed on the Switch the CLIs provide a consolidated view of reboot cause as shown below.
-* Each DPU will update its reboot cause history in the Switch ChasissStateDB upon boot up. The recent reboot-cause can be derived from that list of reboot-causes.
-* The switch side PMON will copy this into the stateDB so that the existing workflow will not be affected.
-* The get_reboot_cause API will return the current reboot-cause of the module.
-
-#### REBOOT_CAUSE DB schema
-```
-Key: "REBOOT_CAUSE|2023_06_18_14_56_12"
+#### 3.4.1 Reboot Cause CLIs
+* There are two existing CLIs "show reboot-cause" and "show reboot-cause history"
+* These two CLIs are extended to "show reboot-cause all" and "show reboot-cause history \<option\>", where the "option" could be DPUx, all or SWITCH
+* When each DPU turns online the NPU chassisd will fetch the reboot-cause using the "get_reboot_cause()" API.
```

Comment:
Why do we need to duplicate here information from `#### Reboot workflow` section?

Reply:
@oleksandrivantsiv There is some overlap mainly to give a quick summary for the convenience of people just reading only CLI section.  Cleaned it.